gooddata-sdk/gooddata_sdk/catalog/export/service.py

```diff
@@ -21,8 +21,35 @@
 
 
 class ExportService(CatalogServiceBase):
+    """
+    ExportService provides the ability to export PDF and Tabular data from GoodData Dashboards.
+    Attributes:
+        _entities_api:
+            A reference to the entities_api of GoodDataApiClient instance.
+        _actions_api:
+            A reference to the actions_api of GoodDataApiClient instance.
+    Methods:
+        _get_exported_content:
+            A static method that gets the exported content using the provided `get_func`.
+        _create_export:
+            A static method that creates an export and returns its ID.
+        _dashboard_id_exists:
+            Checks if the given dashboard_id exists in the workspace.
+        _export_common:
+            Common export method for handling the exports of PDF and Tabular.
+        export_pdf:
+            Export a PDF of a GoodData Dashboard.
+        export_tabular:
+            Export Tabular data from a GoodData Dashboard.
+    """
```

Comment:
Missing **export_tabular_by_insight_id**

Reply:
https://github.com/gooddata/gooddata-python-sdk/pull/298/files/d4ae9dfde8984fe2689b4380eb837a711d4e2e8b#diff-1677d7224ec9ee5fcb82788347dd5058125ffa02c8570f8417b63b86b9dab2fdR277

Reply:
I mean the documentation in the context of service.py.

- ExportService docstring

Reply:
👍